File: learning-ai-platform/test-api.js
const http = require('http');

// 测试后端API连接
const options = {
  hostname: 'localhost',
  port: 4001,
  path: '/api/users/preset-avatars',
  method: 'GET'
};

const req = http.request(options, (res) => {
  console.log(`状态码: ${res.statusCode}`);
  console.log(`响应头: ${JSON.stringify(res.headers)}`);
  
  let data = '';
  res.on('data', (chunk) => {
    data += chunk;
  });
  
  res.on('end', () => {
    console.log('响应数据:', JSON.parse(data));
  });
});

req.on('error', (error) => {
  console.error('请求错误:', error);
});

req.end();

File: learning-ai-platform/test-api.py
import requests

url = 'http://localhost:4001/api/users/preset-avatars'

print('Testing API connection to:', url)

try:
    response = requests.get(url, timeout=5)
    print('Response status code:', response.status_code)
    print('Response headers:', response.headers)
    if response.status_code == 200:
        print('Response body:', response.json())
    else:
        print('Response text:', response.text)
except Exception as e:
    print('Error:', e)